examples/peewee/app.py

```diff
@@ -1,6 +1,8 @@
 from flask import Flask
+from datetime import datetime
 
 import peewee
+from playhouse.signals import pre_save, Model
```

Comment:
Playhouse is the collection of addons that are installed along peewee.
http://peewee.readthedocs.org/en/latest/peewee/playhouse.html

Le mer. 12 août 2015 à 14:48, Bruno Rocha <notifications@github.com> a
écrit :

> In examples/peewee/app.py
> <https://github.com/flask-admin/flask-admin/pull/982#discussion_r36855071>
> :
>
> >
> >  import peewee
> > +from playhouse.signals import pre_save, Model
>
> what is playhouse?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/flask-admin/flask-admin/pull/982/files#r36855071>.
>